apps/meteor/tests/end-to-end/api/livechat/04-dashboards.ts

```diff
@@ -101,6 +293,39 @@ describe('LIVECHAT - dashboards', function () {
 					);
 				});
 		});
+		(IS_EE ? it : it.skip)('should return data with correct values', async () => {
+			const start = moment().subtract(1, 'days').toISOString();
+			const end = moment().toISOString();
+
+			const result = await request
+				.get(api('livechat/analytics/dashboards/chats-totalizers'))
+				.query({ start, end, departmentId: department._id })
+				.set(credentials)
+				.expect('Content-Type', 'application/json')
+				.expect(200);
+
+			const expected = [
+				{ title: 'Total_abandoned_chats', value: 1 },
+				{ title: 'Avg_of_abandoned_chats', value: '14%' },
```

Comment:
If another test before this one adds another room, won't this value change?

Reply:
We're filtering by departments here, which department was only created for this test-case. Hence another tests won't have any impact of this